backend/cmd/headlamp.go

```diff
@@ -370,21 +398,11 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 	if len(contexts) == 0 {
 		log.Println("No contexts/clusters configured by default!")
 	} else {
-		for i := range contexts {
-			context := &contexts[i]
-			proxy, err := config.createProxyForContext(*context)
+		for _, context := range contexts {
+			context := context
```

Comment:
If you are referring to the `context := context` part, it is required because of the Implicit memory aliasing in for loop [Reference](https://husni.dev/beware-of-implicit-memory-aliasing-in-go-foor-loop/).